pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
Actually it exact example.

Do you think it should be {{vagrant_box}} or anything like this?